gulpfile.js

```diff
@@ -800,6 +801,22 @@ function performBuild(watch) {
   });
 }
 
+/**
+ * @param {boolean} compiled
+ */
+function checkBinarySize(compiled) {
+  const file = compiled ? './dist/v0.js' : './dist/amp.js';
+  const size = compiled ? '75.1kB' : '332.3kB';
+  const cmd = `npx bundlesize -f "${file}" -s "${size}"`;
+  log(green('Running ') + cyan(cmd) + green('...\n'));
+  const p = exec(cmd);
```

Comment:
This will result in a warning being printed to advertise the tool's GH integration feature. Not sure it's worth trying to silence it.

https://travis-ci.org/ampproject/amphtml/jobs/362305001#L663-L666

Reply:
https://github.com/siddharthkp/bundlesize/issues/158 I think it's harmless.